.github/workflows/screenshots.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - name: Cache browser download
         uses: actions/cache@v3
         with:
-          key: puppeteer-cache-${{ runner.os }}
+          enableCrossOsArchive: true
```

Comment:
No impact, we still keep the caches separate. We're only actually using `enableCrossOsArchive` to compress/extract using GNU tar (I think) rather than the one that ships with Windows

We want the faster speed mainly, although it does also give the same hashes regardless of OS

Removing `runner.os` isn't an option anyway as our symbolic links (npm workspaces) are incompatible

Reply:
(For context I did try `npm ci` in Ubuntu then restoring in Windows, but sadly no can do)

Reply:
Comments added in:

* https://github.com/alphagov/govuk-frontend/pull/4012

Plus a spike to see how shareable our `node_modules` currently are:

* https://github.com/alphagov/govuk-frontend/pull/4013